main/static/src/style/style.less

```diff
@@ -3,6 +3,9 @@
 @import "../../ext/lesshat/build/lesshat-prefixed";
 @import "../../ext/nprogress/nprogress";
 
+@import "../../ext/bootswatch/flatly/variables";
+@import "../../ext/bootswatch/flatly/bootswatch";
+
```

Comment:
@lipis 
What if you did something like this?
```
@theme: "flatly";
@import "themes/@{theme}/variables.less";
@import "themes/@{theme}/bootswatch.less";
```

Are you planning on having a way to choose one of the bootswatch themes from the admin area?